FAQs.md

```diff
@@ -1,113 +1,135 @@
+- Lockscreen controls and artwork support
+- Fluid navigation
 
-Jockey also contains basic features expected in a music player which include;
 
-- playlist editing
-- lockscreen controls and artwork
-- fluid navigation between all of your content
+3. **How do I create a playlist?**
 
+To create a playlist, navigate to the playlist tab by swiping left or right.
+Once there, click the plus (+) sign below and two options will pop up: "Playlist", and "Smart Playlist".
+Click the plus sign beside playlist, input the playlist name, and click on "create".
 
 
-3. **How do I create a create a playlist?**
+4. **What is a "smart playlist"?**
 
-To create a playlist on Jockey, navigate to the playlist tab by swiping left or right. On the playlist, tab click the plus (+) sign below and two options will pop up Smart Playlist and Playlist. Click the plus sign beside playlist, input playlist name and click on create.
+Smart playlist (as the name implies) is a playlist that automatically sorts and adds music to the list according to customized rules / set parameters.
 
-4. **What is a smart playlist?**
 
-Smart playlist as the name implies is a playlist that automatically sorts and adds music to the list according to customized rules or parameters set. 
+5. **Can I import / export playlists?**
 
-5. **Can I import and export playlists on Jockey?**
+No, playlists can't be imported or exported. Apologies.
 
-No, playlists can neither be imported nor exported.
 
 6. **How can I add songs to a playlist?**
 
-Songs can be added to a playlist from any of the tabs. Songs, Artists, Albums, Genres. To add a song, simply click the 3 dots beside the song title and select the "Add to Playlist" option and select the desired playlist you wish the song to be added to.
+Songs can be added to playlists from any of the tabs (Songs, Artists, Albums, etc).
+To add a song, simply click the 3 dots beside the song title, select the "Add to Playlist" option, and select the desired playlist for said song.
 
-7. **Can I add highlight multiple songs at once to add a playlist?**
 
-No, the highlight function is not yet implemented on Jockey music player. However, when an artist, album or genre containing multiple songs are added to a playlist all the songs contained are added.
+7. **Can I highlight multiple songs at once to add to a playlist?**
+
+No, this has not been implemented yet. However, when an artist, album, or genre containing multiple songs are added to a playlist, all those songs are added.
+
 
 8. **Can I sort tracks?**
 
-Only tracks contained in a playlist can be sorted
+Only tracks contained in a playlist can be sorted.
+
+
+9. **What does the "include/exclude folder" function do?**
+
+This function allows you to include or exclude a folder from the music library.
+When a folder is excluded, the music files in it won't be listed in the music library.
+Note: Inclusion is the default for all folders. You must specify for a folder to be excluded for it not to be included.
```

Comment:
Nit: should there be a line break after the period in the first sentence?

Reply:
On line 70, there are two sentences. "Note: Inclusion is the default for all folders." and "You must specify for a folder to be excluded for it not to be included." They probably just need to be on different lines to be consistent with the rest of the file.

Reply:
Well no... grammatically they would be kept together as they are directly related. The second sentence is meant to be part of the note :)